csrc/ir/utils.cpp

```diff
@@ -1310,6 +1311,40 @@ MmaOpDetails getMmaOpDetails(
   return details;
 }
 
+namespace {
+std::vector<int> getMmaDimsPositions(MmaOp* mma) {
```

Comment:
```suggestion
std::tuple<int, int, int> getMmaDimsPositions(MmaOp* mma) {
```

Also, add a code comment that explains the return value.